shared/profile/showcase-team-offer/container.js

```diff
@@ -44,13 +48,6 @@ const mergeProps = (stateProps, dispatchProps) => {
 
 export default compose(
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
-  lifecycle({
-    componentWillMount: function() {
-      this.props.teamnames.map(name => {
-        !this.props._teamNameToPublicitySettings.get(name) && this.props.loadPublicitySettings(name)
-        !this.props._teamNameToCanPerform.get(name) && this.props.loadTeamOperations(name)
-      })
-    },
-  }),
+  lifecycle({}),
```

Comment:
this line can just go away now

Reply:
we talked in chat: I think we do still need a lifecycle hook here because we somehow need to catch changes to these states, as the old version did